src/engine/source/kvdb/src/kvdb.hpp

```diff
@@ -0,0 +1,86 @@
+#ifndef _KVDB_H
+#define _KVDB_H
+
+#include <string>
+#include <unordered_map>
+#include <vector>
+
+#include "rocksdb/db.h"
+#include "rocksdb/utilities/transaction_db.h"
+
+#define DEFAULT_CF_NAME "default"
```

Comment:
done like this `constexpr static char* DEFAULT_CF_NAME {"default"};`
can this be modify in order to avoid warnings on compile time?
```
warning: ISO C++ forbids converting a string constant to ‘char*’ [-Wwrite-strings]
   71 |     constexpr static char* DEFAULT_CF_NAME {"default"};

```